Sources/SwiftNetKit/Protocols/RequestProtocol.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  RequestProtocol.swift
+//
+//
+//  Created by Sam Gilmore on 7/16/24.
+//
+
+import Foundation
+
+enum MethodType {
+    case get
+    case post
+    case delete
+    case put
+    case patch
+    
+    var stringValue: String {
+        switch self {
+        case .get:
+            return "GET"
+        case .post:
+            return "POST"
+        case .delete:
+            return "DELETE"
+        case .put:
+            return "PUT"
+        case .patch:
+            return "PATCH"
+        }
+    }
```

Comment:
Instead of a defined computed var, you can have `MethodType` conform to `RawRepresentable` and `String`, and each case will just look like `case get = "GET"`.  Then you can reference the string value via `MethodType.get.rawValue`